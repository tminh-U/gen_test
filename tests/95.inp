13
6 12 3 6 1 5 20 2 4 19 19 19 12 
7
7 10
2 9
7 7
4 12
13 13
4 10
7 8
