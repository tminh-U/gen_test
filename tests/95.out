9
25
2
93
2
27
3
